client/src/components/Dashboard.js

```diff
@@ -126,10 +126,20 @@ class Dashboard extends Component {
       { header: "To Language", sortKey: "to_language_name" },
       { header: "Status", sortKey: "status" }
     ];
+    const date = new Date();
+    const hours = date.getHours();
+    let timeOfDay;
+    if (hours < 12) {
+      timeOfDay = "Morning";
+    } else if (hours >= 12 && hours < 17) {
+      timeOfDay = "Afternoon";
+    } else {
+      timeOfDay = "Evening";
+    }
```

Comment:
Can you extract all this logic in a different function?

Reply:
it is done